client-js/app/elements/labrad-app.html

```diff
@@ -52,6 +52,14 @@
       width: 100%;
       height: 100%;
     }
+
+    #location {
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      overflow: hidden;
+      direction: rtl;
+    }
+
```

Comment:
Nit: We have inconsistent whitespacing between css selectors. Most of the other files have no newlines between selectors.